src/test/java/jp/co/soramitsu/sora/util/AllTypesPojo.java

```diff
@@ -0,0 +1,27 @@
+package jp.co.soramitsu.sora.util;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import jp.co.soramitsu.sora.crypto.MapSerializable;
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+public class AllTypesPojo implements MapSerializable {
```

Comment:
the class does not include floating-point numbers. is this done deliberately?

Reply:
Fixed